doc/user_guide/interactions.rst

```diff
@@ -562,11 +562,11 @@ For instance, using our example from above a dropdown can be used to highlight c
     )
 
 
-    
 
-The above example shows all three elements at work. The :input_dropdown: is :bind: to the :selection: which is called from the :condition: encoded through the data. 
 
-The following are the input elements supported in vega-lite: 
+The above example shows all three elements at work. The :input_dropdown: is :bind: to the :selection: which is called from the :condition: encoded through the data.
```

Comment:
It seems like this is not rendering properly in the docs 
![image](https://user-images.githubusercontent.com/4560057/202818504-7781c1e0-7566-428d-8355-91aa939a93bb.png)